### DISCUSSION TITLE
Is there any way to fuzzy search

### DISCUSSION THREAD

**Discussion by indahud** [supabase/supabase#1494] [Questions] [answered]
I am using `textSearch()`. It is required to enter the exact same word in the row in order to get the results. Is there any way to get all the rows even it matches first 2-3 characters.  

```JavaScript
.textSearch('catchphrase', `exam`, { 
    config: 'english' 
  })
```
Should return all the rows with example.

**Comment by easy-street** [comment, marked as answer]
a couple options:

1) `:*` specifies prefix matching with `textSearch`:
```
.textSearch('catchphrase', 'exam:*', {
  config: 'english',
})
```

2) pattern matching with the `like` (case sensitive) and `ilike` (case insensitive) methods:
```
.ilike('catchphrase', 'exam%')
```

**Comment by saendu** [comment]
use type: 'websearch',
